mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs

```diff
@@ -64,10 +68,15 @@ lazy_static! {
     /// This approach addresses the `replacement transaction` issue, which occurs when different transactions share the same nonce.
     pub static ref MM_CTX1: MmArc = MmCtxBuilder::new().with_conf(json!({"use_trading_proto_v2": true})).into_mm_arc();
     pub static ref GETH_WEB3: Web3<Http> = Web3::new(Http::new(GETH_RPC_URL).unwrap());
-    pub static ref SEPOLIA_WEB3: Web3<Http> = Web3::new(Http::new(SEPOLIA_RPC_URL).unwrap());
     // Mutex used to prevent nonce re-usage during funding addresses used in tests
     pub static ref GETH_NONCE_LOCK: Mutex<()> = Mutex::new(());
+}
+
+#[cfg(any(feature = "sepolia-maker-swap-v2-tests", feature = "sepolia-taker-swap-v2-tests"))]
+lazy_static! {
+    pub static ref SEPOLIA_WEB3: Web3<Http> = Web3::new(Http::new(SEPOLIA_RPC_URL).unwrap());
     pub static ref SEPOLIA_NONCE_LOCK: Mutex<()> = Mutex::new(());
```

Comment:
does this use any docker instance? why don't we move such tests outside docker (to test_inner)?

Reply:
Hmm, yes, the Sepolia code doesn’t use Geth-generated addresses.  
We could move the code to the `sepolia_tests` test crate.  
However, there’s still a chance that when we update the Swap V2 smart contracts, for example add `burnFee` to the taker swap contract, this contract could start working on the Geth node (we faced similar issue during the NFT maker swap V2 implementation and testing). In that case, the code could be moved back to docker_tests.

Let’s plan this when we update the taker swap v2 contract, if it doesn’t work on Geth, we’ll move taker/maker Sepolia code to a separate test crate.